<?xml version="1.0"?>
<launch>
    <include file="$(find face_detection)/launch/webcam.launch" />
    <include file="$(find legacy_measurement)/launch/legacy_measurement.launch">
        <arg name="topic" value="/face_detection/forehead" />
    </include>
</launch>
